cove_360/lib/threesixtygiving.py

```diff
@@ -108,10 +109,24 @@ def common_checks_360(context, upload_dir, json_data, schema_obj):
     common_checks = common_checks_context(upload_dir, json_data, schema_obj, schema_name, context)
     cell_source_map = common_checks['cell_source_map']
 
+    validation_errors = context['validation_errors']
+    validation_errors_grouped = defaultdict(list)
```

Comment:
Would it not be better to extract this to a new function and call it in this one?

Reply:
Yes!